### ISSUE TITLE
ParmEd Error with AmberTools ==20.0

### ISSUE THREAD

**Issue by SimonBoothroyd** [choderalab/openmoltools#301]
The new version of `ambertools ==20.0` seems to have broken compatibility with `openmoltools`.

In particular, the version number of the ParmEd which ships with the new ambertools seems to be incorrect.

ambertools = 20.0:
`parmed                    at20RC5+54.g5702a232fe.dirty          pypi_0    pypi`

ambertools = 19.11:
`parmed                    3.2.0                    pypi_0    pypi`

which yields the following exception:

```
# If ParmEd is older than 2.0.4 then halt - newer ParmEd is required to have correct FudgeLJ/FudgeQQ retained in GROMACS topologies.
    try:
        ver = parmed.version
    except:
        oldParmEd = Exception('ERROR: ParmEd is too old, please upgrade to 2.0.4 or later')
        raise oldParmEd
    if ver < (2,0,4):
>       raise RuntimeError("ParmEd is too old, please upgrade to 2.0.4 or later")
E       RuntimeError: ParmEd is too old, please upgrade to 2.0.4 or later
```

cc @jaimergp

**Comment by jchodera** [comment]
This is fixed in #274. I'll cut a new release now.

**Comment by jchodera** [comment]
cc https://github.com/conda-forge/ambertools-feedstock/issues/35
